cmake/CMakeLists.txt

```diff
@@ -122,6 +119,11 @@ else()
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /d2FH4-")
     set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /d2FH4-")
   endif()
+  if(WINDOWS_STORE)
+    # cmake/external/protobuf/src/google/protobuf/compiler/subprocess.cc and onnxruntime/core/platform/windows/env.cc call a bunch of Win32 APIs.
+    # For now, we'll set the API family to desktop globally to expose Win32 symbols in headers; this must be fixed!
```

Comment:
is there a task tracking the "this must be fixed!" part?

Reply:
This is on the ORT side. Changming is aware of it: https://github.com/microsoft/onnxruntime/pull/3126#discussion_r393834602